### ISSUE TITLE
Inquiry about model initialization

### ISSUE THREAD

**Issue by QiangweiPeng** [GuangyuWangLab2021/cellDancer#6]
Dear GuangyuWangLab members,

When I use the cellDancer, I noticed that the initialization of the neural network is done by loading the saved parameters. Could you please share how these parameters are generated? Because the initialization of the model might affect the direction of the final RNA velocity.

**Comment by Abclisy** [comment]
Thank you for reaching out to us. To optimize the training time, we offer two pre-trained models. The first model is pre-trained on gene [*Sulf2*](https://www.nature.com/articles/s41587-023-01728-5/figures/1) in mouse pancreatic endocrinogenesis. The second model is pre-trained on gene [*Ntrk2*](https://www.nature.com/articles/s41587-023-01728-5/figures/3) in mouse hippocampal dentate gyrus during neurogenesis.

**Comment by QiangweiPeng** [comment]
Thank you for your reply! This is indeed a reasonable option to optimize training time. However, this doesn't seem to ensure that the RNA velocity is in the right direction. For example, the value of the loss function for the direction exactly opposite to the correct direction is also very low. And when the initial direction is closer to the wrong opposite direction, the model will converge to the wrong direction.

**Comment by QiangweiPeng** [comment]
Yes, I agree that convergence to the global optimum is difficult. My confusion is that in the current setup the exact opposite wrong direction and the exact correct direction both seem to be global optimum?

**Comment by Abclisy** [comment]
In our model, although each cell has its own specific kinetic rates, the rates are not totally independent. They are predicted from the same neural network. We didn't observe the scenario that all cells have the exact opposite wrong direction at the same time during testing cellDancer on both simulation datasets and five case studies. We are happy to discuss more if you get the opposite direction of all cells in your datasets.

**Comment by QiangweiPeng** [comment]
Ok, got it. cellDancer is indeed performing very well, I haven't found the exact opposite direction yet. Thank you very much for your patient reply.
